### ISSUE TITLE
Save an image before sending to svd

### ISSUE THREAD

**Issue by KirillSamoilenko** [city96/ComfyUI_NetDist#10]
hey! So my workflow is:

1 - sdxl txt2img in high fidelity and detail in, lets say landscape 1344x768
2 - resize down to 1024x576 for svd
3 - img2vid from that image

using this i am able to get png sequence of img2vid out for all gpus involved but i also want to save the high fidelity image before sending to svd and current implementation of fetch remote messes up this step. Am I right to think that this is part of whats mentioned in roadmap and for now if i need to save images in-between then i can only do it for main comfyui instance?

**Comment by city96** [comment]
>Am I right to think that this is part of whats mentioned in roadmap

Yup. Fetching multiple different images per workflow would let you return both the final one as well as your high res input ones, which you could then save locally on your main instance. No ETA yet, but I'll get to it.

>and for now if i need to save images in-between then i can only do it for main comfyui instance?

As a temp fix you could try and remove `"SaveImage"` from [this line](https://github.com/city96/ComfyUI_NetDist/blob/main/core/dispatch.py#L91). That way the images still get saved on your main/secondary instance.
